tests/js/firstDiscoveryEditorTests.js

```diff
@@ -284,6 +284,13 @@ https://github.com/gpii/universal/LICENSE.txt
         });
     };
 
+    gpii.tests.firstDiscovery.testDefaultLocale = function (that) {
+        var locale = that.prefsEditorLoader.options.defaultLocale;
+        var expectedLocale = fluid.get(fluid.defaults("gpii.firstDiscovery.schemas.language"), ["schema", "properties", "gpii.firstDiscovery.language", "default"]);
```

Comment:
The right hand side is exactly the formula in the expander that you wanna test. If both are returning same wrong values, the test would still pass. I think it's OK to directly check against the expected string such as "en-US".